lark/utils.py

```diff
@@ -134,7 +151,8 @@ def get_regexp_width(expr):
             raise ImportError('`regex` module must be installed in order to use Unicode categories.', expr)
         regexp_final = expr
     try:
-        return [int(x) for x in sre_parse.parse(regexp_final).getwidth()]
+        # TODO: bug? Returns an int?
+        return [int(x) for x in sre_parse.parse(regexp_final).getwidth()]   # type: ignore[attr-defined]
```

Comment:
`sre_parse.parse(regexp_final).getwidth()` returns an int, which you can't iterate over. Not sure what was intended here.

Reply:
Ahah, bug in typeshed! https://github.com/python/typeshed/blob/cb5b31cf15da14c63cb6467b5a29315471173ced/stdlib/sre_parse.pyi#L68

Reply:
Well, it's quite an obscure feature. Most re libraries in other languages don't even have it.

Reply:
Fixed in typeshed now, but missed today's mypy 0.960 release.

Reply:
Oops, can't resolve until I fix the comment (tomorrow).